README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://GUSILLUS.github.io/layout_calendar/)
+- [TEST REPORT LINK](https://GUSILLUS.github.io/layout_calendar/report/html_report/)
```

Comment:
Please, check your is it correct link

Reply:
Yea, all is good 'TEST REPORT' didn`t work because this task dont have npm test